pyproject.toml

```diff
@@ -37,7 +37,7 @@ classifiers = [
 dependencies = [
     "numpy>=1.23.5",
     "scipy>=1.9.3,<1.13.0",
-    "casadi>=3.6.3",
+    "casadi>=3.6.5",
```

Comment:
I have checked and I can confirm that CasADi v3.6.5 is available on both PyPI and conda-forge distribution channels.